src/pkg/services/m365/api/graph/concurrency_middleware.go

```diff
@@ -193,6 +223,11 @@ func QueueRequest(ctx context.Context) {
 	}
 }
 
+func ResetLimiter(ctx context.Context) {
```

Comment:
Can you add a comment explaining how this affects the limiter, and when it should reasonably be used?